clang/lib/AST/Decl.cpp

```diff
@@ -5042,7 +5042,7 @@ void RecordDecl::completeDefinition() {
 
   // Layouts are dumped when computed, so if we are dumping for all complete
   // types, we need to force usage to get types that wouldn't be used elsewhere.
-  if (Ctx.getLangOpts().DumpRecordLayoutsComplete)
+  if (Ctx.getLangOpts().DumpRecordLayoutsComplete && !isDependentType())
```

Comment:
Can you update the comment above to reflect this change.

Reply:
Sure; I was going to do that but I felt it wasn’t really ‘necessary’, but it can’t hurt.